package.json

```diff
@@ -23,22 +23,25 @@
     "README.md",
     "LICENSE.txt",
     "CHANGELOG.md",
+    "index.d.ts",
```

Comment:
Is it worth adding a `types` key to this file? While it's not required for `/index.d.ts`, [TypeScript's docs still recommend it](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package), it seems.

Reply:
Oh I didn't know - thanks for the tip!